src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1521,9 +1517,9 @@ private String init(boolean initFull) {
                             setHasRsyncScript(false);
                         }
                     } catch (RuntimeException ex) {
-                        logger.warning("Problem getting rsync script: " + ex.getLocalizedMessage());
+                        logger.warning(()->"Problem getting rsync script: " + ex.getLocalizedMessage());
                     } catch (CommandException cex) {
-                        logger.warning("Problem getting rsync script (Command Exception): " + cex.getLocalizedMessage());
+                        logger.warning(()->"Problem getting rsync script (Command Exception): " + cex.getLocalizedMessage());
```

Comment:
What do these changes do? It looks like it just creates clutter by adding anonymous functions. Is there a benefit?

Reply:
There's a huge benefit in not creating the string unless it's actually going to be logged. Note that creating strings requires O(n) time and contiguous space, which means that the memory management and garbage collector have to work harder.
NetBeans actually displays a warning and offers an auto-fix when you use the previous version.